### PR TITLE
Rebase on external access level

### LINKED ISSUE #230: Changes to External Access field
This currently maps to the core license field, but needs to be replaced by a selector of the following values:

* Not available
* Direct access
* Public use
* Licensed use
* Data enclave
* Open access

As these are not proper licenses is best to create a new separate field `external_access` for this. 
Existing datasets should be updated to set `Not available` on this new field (most don't define a license currently https://ridl.unhcr.org/api/action/package_search?facet=true&facet.field=[%22license_id%22]&rows=0)

### PR DESCRIPTION
**Pull request by roll**
- fixes #230

### CODE REVIEW

File: ckanext/unhcr/templates/snippets/license.html [hunk 1]
```diff
@@ -1,5 +1,6 @@
-{% ckan_extends %}
-
-{% block license_title %}
-  <h2 class="module-heading"><i class="fa fa-medium fa-lock"></i> {{ _('External access level') }}</h2>
-{% endblock %}
+<section class="module module-narrow module-shallow license">
```
Comment (amercader) [inline ckanext/unhcr/templates/snippets/license.html]: Why do we need a custom snippet and helper here? Can't we use the standard `select` snippet?

Comment (roll) [reply ckanext/unhcr/templates/snippets/license.html]: @amercader 
It's for `package_read` sidebar

**Comment by roll** [comment]
@amercader 
It's ready for a review

File: ckanext/unhcr/templates/snippets/license.html (hunk 1 above)
Comment (amercader) [reply ckanext/unhcr/templates/snippets/license.html]: sorry for the noise, I need to pay more attention

**Comment by amercader** [comment]
Given that `external_access_level` is a required field, we need to run a migration to provide a default value on all existing datasets. Otherwise if you edit the resources of an existing dataset you get an error, because CKAN internally calls package_update to add/edit resources and this fails with the following:

Screenshot_2019-10-28 Edit - Unnamed resource - Test data collector - RIDL

We just need to update all existing datasets once to have `externa_access_level=not_specified`. We can adapt the script in `scripts/update_private_datasets.py` for this.

**Comment by roll** [comment]
@amercader 
Oh I see I'll add the script

**Comment by roll** [comment]
@amercader 
I have added the script

**Comment by roll** [comment]
Adria, please test the script if you'd like. I'm merging to simplify the `ckan-2.7` branch creation

**Pull request merged**